src/azure-cli/azure/cli/command_modules/eventhubs/tests/latest/test_cluster_commands.py

```diff
@@ -32,42 +32,39 @@ def test_eh_cluster(self):
             'tier': 'Standard'
         })
 
-        # create Resource group
-        self.cmd('az group create --resource-group {rg} --location {loc1}')
-
         # Create Cluster
-        self.cmd('eventhubs cluster create --resource-group {rg} --name {clustername} --location {loc} --tags tag1=value1',
+        self.cmd('eventhubs cluster create --resource-group test-migration --name {clustername} --location eastus --tags tag1=value1',
```

Comment:
@evelyn-ys, we will need to do it for this test case because cluster is an expensive resource to create and recently our testscripts created too many clusters that were unmonitored and cost us a lot and hence we decided to use a resource group that we can clean up every day to avoid wastage of resources and money.

Reply:
The resource group preparation will do auto clean after test finishing. I don't understand why there will be "too many clusters that were unmonitored". Won't these clusters being deleted during deleting resource group?

Reply:
Because Cluster can be deleted only 4 hours after creation.

Reply:
Then can you add comments to the test for others who might need to rerun tests after your change to know why and how to prepare for the test

Reply:
I have added the comments in cluster test file.

Reply:
Can this PR be merged if this comment appears good?